vendor/github.com/bitrise-steplib/bitrise-step-build-router-start/bitrise/bitrise.go

```diff
@@ -220,6 +243,108 @@ func (app App) StartBuild(workflow string, buildParams json.RawMessage, buildNum
 	return response, nil
 }
 
+// GetBuildArtifacts ...
+func (app App) GetBuildArtifacts(buildSlug string) (buildArtifactsResponse BuildArtifactsResponse, err error) {
```

Comment:
Does this belong in the bitrise-step-build-router-start step?

Reply:
I think it's fine here. @trapacska WDYT?